forge-gui/res/cardsfolder/upcoming/vraska_betrayals_sting.txt

```diff
@@ -0,0 +1,15 @@
+Name:Vraska, Betrayal's Sting
+ManaCost:4 B PB
+Types:Legendary Planeswalker Vraska
+Loyalty:6
+K:Compleated
+A:AB$ Draw | Cost$ AddCounter<0/LOYALTY> | Planeswalker$ True | SubAbility$ DBLoseLife | SpellDescription$ You draw a card and you lose 1 life. Proliferate.
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ 1 | SubAbility$ DBProliferate 
+SVar:DBProliferate:DB$ Proliferate 
+A:AB$ Animate | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | ValidTgts$ Creature | TgtPrompt$ Select target creature | RemoveAllAbilities$ True | Colors$ Green | Types$ Artifact,Treasure | Abilities$ TreasureSac | RemoveCreatureTypes$ True | RemoveCardTypes$ True | Duration$ Permanent | SpellDescription$ Target creature becomes a Treasure artifact with "{T}, Sacrifice this artifact: Add one mana of any color" and loses all other card types and abilities.
```

Comment:
You shouldn't need RemoveCreatureTypes

The engine should handle that rules part already